celerylogger/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-buster
```

Comment:
Could image `python:3.7-slim-buster` do the job? It's WAY smaller than this one.

Reply:
Probably! I'll test it out, thanks!

Reply:
It works, thanks!